test/Dapr.Client.Test/InvokeApiTest.cs

```diff
@@ -5,10 +5,14 @@
 
 namespace Dapr.Client.Test
 {
+    using System;
     using System.Collections.Generic;
     using System.Net;
+    using System.Text;
     using System.Text.Json;
     using System.Threading.Tasks;
+    using System.Runtime.Serialization.Formatters.Binary;
```

Comment:
What's this being used for? This sets off some alarm bells.

Reply:
Unused.